### ISSUE TITLE
Non-admin do not have permission to search for Categories in Cart Price Rules

### ISSUE THREAD

**Issue by heldchen** [magento/magento2#6168]
non-admin user roles with access rights to "Products*", "Marketing", "Promotions", "Cart Price Rules" cannot browse for categories in `Rule Action > Category Condition`. the ajax url `/admin/catalog_rule/promo_widget/chooser/attribute/category_ids/form/sales_rule_form/?isAjax=true` returns "permission denied"
### Preconditions
1. Magento 2.1.0
### Steps to reproduce
1. Create Role with the access rights "Products*", "Marketing", "Promotions", "Cart Price Rules"
2. Assign Role to user
3. use this user to create new Cart Price Rule
4. in this new rule under Actions > Apply the rule ... add condition "Product Attribute: Category"
5. click on "..."
6. click on the "Open Choser" icon to browse for a category
### Expected result
1. the category tree should be loaded
### Actual result
1. the ajax request does not return the category tree but an error

**Comment by heldchen** [comment]
here's a list of the actual permissions set in the role as stored in the db:

```
Magento_Backend::admin
Magento_Backend::content
Magento_Backend::content_elements
Magento_Backend::marketing
Magento_Backend::marketing_communications
Magento_Backend::marketing_user_content
Magento_Backend::stores
Magento_Backend::stores_attributes
Magento_Catalog::attributes_attributes
Magento_Catalog::catalog
Magento_Catalog::catalog_inventory
Magento_Catalog::categories
Magento_Catalog::products
Magento_Catalog::update_attributes
Magento_CatalogRule::promo
Magento_Cms::block
Magento_Cms::page
Magento_Cms::save
Magento_Newsletter::subscriber
Magento_Review::reviews_all
Magento_SalesRule::quote

```

**Comment by katmoon** [comment]
Hi @heldchen 

Thank you for reporting the issue. It is reported as `MAGETWO-56869` internally.

Thanks,
Kateryna.

**Comment by slavvka** [comment]
The issue has been fixed and delivered to mainline

**Comment by Ctucker9233** [comment]
@slavvka is this fix safe to integrate into a 2.1.1 production environment or is there a patch pending?
